pkg/app/pagewriter/sign_in_page_test.go

```diff
@@ -18,10 +20,11 @@ import (
 var _ = Describe("SignIn Page", func() {
 
 	Context("SignIn Page Writer", func() {
+		var request *http.Request
 		var signInPage *signInPageWriter
 
 		BeforeEach(func() {
-			errorTmpl, err := template.New("").Parse("{{.Title}}")
+			errorTmpl, err := template.New("").Parse("{{.Title}} | {{.RequestID}}")
```

Comment:
Do we need to test the error page rendering from the sign in page tests? I would have thought the error page tests would be sufficient? Or is there something you're concerned about with the sign in page breaking the request ID handling?

Reply:
I did change the sign-in page method signature slightly to take in the `*http.Request` now where it didn't before - just for its error scenario where it had a problem and made the error page (which needed the `req` for the `RequestID`.

I figured it was worth testing since I altered the codepath?

Reply:
Ok that makes sense